bin.src/addSeds.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import os
+import argparse
+import warnings
+import numpy as np
+from lsst.sims.movingObjects import Orbits
+from itertools import repeat
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description="Add SEDs to a series of orbits.")
+    parser.add_argument("--orbitFile", type=str, default=None,
```

Comment:
The script will work with any orbit file that the 'generate coefficients' script (i.e. the same as the underlying Orbit class, but with a little bit of parsing to fix the "is this one comment line or two"?). The "tmp" stuff is problematic though, as I think this is actually three columns? 

The header should have a header column for each data column (because underneath it all, it's pandas reading the data). So the space between !! and the first column name is problematic, but if you join them together it's ok.

I think if you look at the des files you actually ended up using with the generateCoefficients script, you might find slightly different header lines? (because this header woudln't work with generateCoefficients either I don't think).